aws/resource_aws_backup_plan_test.go

```diff
@@ -338,6 +384,48 @@ resource "aws_backup_plan" "test" {
 `, stringID, stringID, stringID)
 }
 
+func testAccBackupPlanWithLifecycleDeleteAfterOnly(stringID string) string {
+	return fmt.Sprintf(`
+resource "aws_backup_vault" "test" {
+  name = "tf_acc_test_backup_vault_%s"
+}
+
+resource "aws_backup_plan" "test" {
+  name = "tf_acc_test_backup_plan_%s"
+
+  rule {
+    rule_name          = "tf_acc_test_backup_rule_%s"
+    target_vault_name  = "${aws_backup_vault.test.name}"
+    schedule           = "cron(0 12 * * ? *)"
+    lifecycle {
+        delete_after       = "7"
```

Comment:
Nit: Formatting here and below

```suggestion
        delete_after = "7"
```